test/cors-gate.spec.js

```diff
@@ -126,6 +126,20 @@ describe('cors-gate', function() {
         .get('/get')
         .expect(403, done);
     });
+
+    it('can be given a function', function(done) {
+      this.app.use(corsGate({
+        strict: true,
+        allowSafe: () => false,
```

Comment:
probably test `() => true` too, right?

Reply:
We could. I figured the logic of `allowSafe` itself is well tested - we just want to ensure that it supports calling a function in the first place.